shopping_mall/backend/ai/embeddings.py

```
@@ -0,0 +1,67 @@
+    if provider == "openrouter":
+        from chromadb.utils.embedding_functions import OpenAIEmbeddingFunction
+        model = settings.embed_model or "openai/text-embedding-3-small"
+        # PRIMARY_LLM_API_KEY를 기본으로 재사용 — 별도 키 불필요
+        api_key = settings.embed_api_key or settings.primary_llm_api_key
+        return OpenAIEmbeddingFunction(
+            api_key=api_key,
+            model_name=model,
+            api_base=_OPENROUTER_BASE_URL,
+        )
+
+    elif provider == "ollama":
```

Ruff (0.15.10)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)